changelogs/fragments/2567-iam-user-info-fix-get.user-condition.yml

```diff
@@ -0,0 +1,3 @@
+---
+bugfixes:
+  - iam_user_info - Acutally call GetUser when only user name is supplied insteaf od listing and filtering from all users
```

Comment:
```suggestion
  - iam_user_info - Actually call GetUser when only user name is supplied instead of listing and filtering from all users (https://github.com/ansible-collections/amazon.aws/pull/2567).
```